google-cloud-firestore/src/test/java/com/google/cloud/firestore/TransactionTest.java

```diff
@@ -384,6 +630,46 @@ public void getMultipleDocuments() throws Exception {
     assertEquals(commit(TRANSACTION_ID), requests.get(2));
   }
 
+  @Test
+  public void getMultipleDocumentsAsync() throws Exception {
```

Comment:
The above comment also applies here. The code path you are testing is already covered by `getDocumentAsync()`. This also applies to other test functions added below.

Reply:
test removed